pkg/server/api/api_test.go

```diff
@@ -51,6 +59,87 @@ func TestIDFromProto(t *testing.T) {
 	}
 }
 
+func TestFetchAuthEntries(t *testing.T) {
+	ef := &fakeentryfetcher.EntryFetcher{}
```

Comment:
Maybe this can just be initialized down inside the loop? Then it's clear that we're starting from a clean slate instead of having to reset state every time (e.g. the err)

Reply:
I'm removing the FetchAuthEntries public function from the api package.